### PR TITLE
Refactor RSS app

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: src/apps/rss/index.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            return next(err)
+          })
+      }
+    })
+  },
+]
+
+app.get('/explore/feed/rss', ...resolve)
+app.get('/:username/feed/rss', ...resolve)
+app.get('/:username/:slug/feed/rss', ...resolve)
```
Comment (dzucconi) [inline src/apps/rss/index.ts:66]: If this is using react-router; why bother spinning up a new app for it?

Comment (dzucconi) [reply src/apps/rss/index.ts:66]: Oh I see; custom layout; yeah.

Comment (broskoski) [reply src/apps/rss/index.ts:66]: the specific resolve and caching

File: src/v2/pages/rss/ChannelRss/index.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import {
+  ChannelRss as ChannelRssType,
+  ChannelRssVariables,
+} from '__generated__/ChannelRss'
+import { RssBlock } from '../components/RssBlock'
+import RssLayout from '../components/RssLayout'
+import channelRssQuery from './queries/channelRss'
+
+interface ChannelRssProps {
+  params: any
```
Comment (dzucconi) [inline src/v2/pages/rss/ChannelRss/index.tsx:12]: Shouldn't this just be `params: { slug?: string }`?

**Review by dzucconi — APPROVED** [review]
Cool

**Pull request merged**